client/slack.go

```diff
@@ -123,9 +124,17 @@ func newMessageEvent(e *s.MessageEvent, u *s.User) messageEvent {
 		Message:         e.Text,
 		Timestamp:       e.Timestamp,
 		ThreadTimestamp: e.ThreadTimestamp,
+		Thread: getThread(e),
```

Comment:
What is the purpose of this field? Do we need extra field? I guess this is so if you want to reply in thread you don't have to worry about checking ThreadTimestamp for nil.

I think it might just make sense to set ThreadTimestamp according to the logic in the getThread().

Reply:
@pamelin  I did not want to break existing flows, but yes you are right we can even populate ThreadTimestamp the way that it will be by default the thread_ts otherwise ts.

I would like to avoid making this check all the time in flows.

Reply:
@balazs-fark-epam I agree with Paul, and do not forget to format this one too.